content/docs/azure/authentication.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Restricting Access & Authentication "
```

Comment:
Recommendation: Reword the title, as it currently implies that we're restricting auth as well as restricting access.